### ISSUE TITLE
Cannot generate Gantt chart

### ISSUE THREAD

**Issue by wenlzhang** [tomoyukim/vscode-mermaid-editor#20]
The Gantt chart I created can be previewed correctly, both in this VS Code plugin and in other mermaid tools. However, whenever I try to export/generate the graph, it gives the following errors:
```
Parse error on line 1:
#mermaid-16011334473
^
Expecting 'NEWLINE', 'SPACE', 'GRAPH', got 'BRKT'
```

**Comment by tomoyukim** [comment]
@wenlzhang Hi, thank you for reporting the symptom. Could you share your mmd code if possible? It's really helpful to reproduce and investigate the buggy behavior.

**Comment by wenlzhang** [comment]
After some further checking, i.e. when trying to simplify the Gantt diagram I have to be able to post it here, it turns out it can be some encoding issue for a pasted symbol from Word... The pasted symbol is `‐`. However, the correct one should be `-`, which is extremely tricky to notice in this case...

From the error message, it says that the error happens in line 1, but it actually happens in line 8. For example, the mentioned error can be reproduced from the following example:
```
gantt
    title A Gantt Diagram
    dateFormat  YYYY-MM-DD
    section Section
    A task           :a1, 2014-01-01, 30d
    Another task     :after a1  , 20d
    section Another
    Task-in‐sec (The first dash symbol is correct while the second one is wrong)      :2014-01-12  , 12d
    another task      : 24d
```

**Comment by tomoyukim** [comment]
Thank you for the information! I found the following error is occurred during generation. I'll fix it in the next version.
`Failed to execute 'btoa' on 'Window': The string to be encoded contains characters outside of the Latin1 range.
`

**Comment by tomoyukim** [comment]
Thank you for waiting. This should be resolved in v0.9.0.
